packages/main/src/Icon.js

```diff
@@ -181,7 +181,7 @@ class Icon extends UI5Element {
 
 	static async onDefine() {
 		this.createGlobalStyle(); // hide all icons until the first icon has rendered (and added the Icon.css)
-		await fetchI18nBundle("@ui5/webcomponents");
+		await fetchI18nBundle("@ui5/webcomponents-icons");
```

Comment:
This change is a good first step, but please don't close the issue as the ui5-icon web component is still hardcoding where the message bundles are coming from.

For a generic solution, we also need to think of a mapping from icon namespace to icon package.

For example, the icon tnt/workflow-editor should look up texts from @ui5/webcomponents/icons-tnt.

I'll add this to the issue.